pkg/pb/contact.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package contact;
+
+import "gogo.proto";
+import "node.proto";
+
+service Contact {
+    rpc PingNode(ContactPingRequest) returns (ContactPingResponse);
+}
+
+service Node {
+    rpc Checkin(CheckinRequest) returns (CheckinResponse);
+}
+
+message CheckinRequest {
+    node.NodeAddress sender = 1;
+    bytes sender_nodeid = 2 [(gogoproto.customtype) = "NodeID", (gogoproto.nullable) = false];
```

Comment:
nit - I think field names `address` and `id` would be sufficient. Feel free to ignore

Reply:
sender_nodeid shouldn't be here, it should be captured from the handshake instead.